.github/workflows/build.yml

```diff
@@ -1,119 +1,81 @@
 # GitHub Actions Workflow is created for testing and preparing the plugin release in the following steps:
-# - validate Gradle Wrapper,
-# - run 'test' and 'verifyPlugin' tasks,
-# - run 'buildPlugin' task and prepare artifact for the further tests,
-# - run 'runPluginVerifier' task,
-# - create a draft release.
+# - Validate Gradle Wrapper.
+# - Run 'test' and 'verifyPlugin' tasks.
```

Comment:
Are these formatter related changes? Might want to standardize it using `.idea/codeStyles`

Reply:
Nope, updated the file based on the IntelliJ plugin template repo